components/infobox/commons/infobox_league.lua

```diff
@@ -482,6 +482,12 @@ end
 ---@param args table
 ---@param links table
 function League:_setLpdbData(args, links)
+	---@param name string
+	---@return string
+	local function pageifyLink(name)
+		return (name:gsub(' ', '_'))
+	end
```

Comment:
> wouldn't it also make sense to have `resolve_redirect` be part of this too, instead of being called as part of the param below?

maybe optional if we move it to a sep module
like in some places we wouldn't want to resolve or already have resolved stuff we want to have underscores instead of spaces